ai-agent/api/routers/sheet_generator.py

```
@@ -0,0 +1,16 @@
+router = APIRouter(prefix="/sheet-generator", tags=["/sheet-generator"])
+
+@router.post("/generate-sheet")
+async def create_information_sheet(profile: UserProfile):
+    """
+    フロントエンドから送られてきたユーザー経歴情報をもとに情報シートを生成するAPIエンドポイント
+    """
+    try:
+        sheet = generate_information_sheet(profile)
+        return {"status": "success", "sheet": sheet}
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Sheet generation failed: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)